ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/index.scss

```
@@ -0,0 +1,22 @@
+.advanced-gas-fee-input {
+  a.advanced-gas-fee-input__edit-link {
+    display: inline;
+    font-size: 12px;
+    padding: 0;
+    white-space: nowrap;
+  }
+
+  .form-field__heading-title > h6 {
+    font-size: 12px;
```

Use design system variable:`$font-size-h7` or mixin: `@include H7` instead of static pixel value
```suggestion
    font-size: $font-size-h7;
```